systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectIsolatedST.java

```diff
@@ -594,6 +606,84 @@ void testSecretsWithKafkaConnectWithTlsAndScramShaAuthentication(ExtensionContex
+        String imageName = Environment.getImageOutputRegistry() + "/" + testStorage.getNamespaceName() + "/connect-" + Util.hashStub(String.valueOf(new Random().nextInt(Integer.MAX_VALUE))) + ":latest";
+
+        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(testStorage.getNamespaceName(), testStorage.getTopicName()).build());
+
+        final Plugin echoSinkPlugin = new PluginBuilder()
+            .withName(echoSinkConnector)
+            .withArtifacts(
+                new JarArtifactBuilder()
+                    .withUrl(ECHO_SINK_JAR_URL)
+                    .withSha512sum(ECHO_SINK_JAR_CHECKSUM)
+                    .build())
+            .build();
+
+        KafkaConnect connect = KafkaConnectTemplates.kafkaConnect(testStorage.getClusterName(), testStorage.getNamespaceName(), testStorage.getNamespaceName(), 1)
+            .editMetadata()
+                .addToAnnotations(Annotations.STRIMZI_IO_USE_CONNECTOR_RESOURCES, "true")
+            .endMetadata()
+            .editOrNewSpec()
+                .addToConfig("key.converter.schemas.enable", false)
+                .addToConfig("value.converter.schemas.enable", false)
+                .addToConfig("key.converter", "org.apache.kafka.connect.storage.StringConverter")
+                .addToConfig("value.converter", "org.apache.kafka.connect.storage.StringConverter")
+                .withNewBuild()
+                    .withPlugins(echoSinkPlugin)
+                    .withNewDockerOutput()
+                        .withImage(imageName)
+                    .endDockerOutput()
+                .endBuild()
+            .endSpec()
+            .build();
+
+        resourceManager.createResource(extensionContext, connect, ScraperTemplates.scraperPod(testStorage.getNamespaceName(), testStorage.getScraperName()).build());
+        LOGGER.info("Deploy NetworkPolicies for KafkaConnect");
+        NetworkPolicyResource.deployNetworkPolicyForResource(extensionContext, connect, KafkaConnectResources.deploymentName(testStorage.getClusterName()));
+
+        Map<String, Object> echoSinkConfig = new HashMap<>();
+        echoSinkConfig.put("topics", testStorage.getTopicName());
+        echoSinkConfig.put("level", "INFO");
+        echoSinkConfig.put("fail.task.after.records", "1");
+
+        LOGGER.info("Creating EchoSink connector");
+        resourceManager.createResource(extensionContext, KafkaConnectorTemplates.kafkaConnector(echoSinkConnector, testStorage.getClusterName())
+            .editOrNewSpec()
+                .withTasksMax(1)
+                .withClassName(ECHO_SINK_CLASS_NAME)
+                .withConfig(echoSinkConfig)
+                .withAutoRestart(new AutoRestartBuilder().withEnabled().build())
+            .endSpec()
+            .build());
+
+        // Send Messages to topic
+        KafkaClients kafkaClients = new KafkaClientsBuilder()
+            .withTopicName(testStorage.getTopicName())
+            .withMessageCount(10)
```

Comment:
Maybe explanation why is it set to 10 or something would be great.

Reply:
added variable with explanation